package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.8.0",
+    "react-redux": "^8.0.5",
```

Comment:
- [ ] Good job working on  Bookstore: add reducers and actions 👍, however, it should be better if structure of the files should follow the `ducks pattern`: 
```` bash
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | store.js
    | index.js
````

Reply:
Okay, @joel-kalema. Thanks for the tip. I've done that now.